TESTING.rst

```diff
@@ -55,8 +55,33 @@ Follow the guidelines when writing unit tests:
   tests, so we run Pytest with ``--disable-warnings`` but instead we have ``pytest-capture-warnings`` plugin that
   overrides ``recwarn`` fixture behaviour.
 
-**NOTE:** We plan to convert all unit tests to standard "asserts" semi-automatically, but this will be done later
-in Airflow 2.0 development phase. That will include setUp/tearDown/context managers and decorators.
+
+.. note::
+
+  We are in the process of convert all unit tests to standard "asserts" and pytest fixtures
+  so if you find some tests that are still using classic setUp/tearDown approach or unittest asserts, feel
+  free to convert them to pytest.
+
+Airflow configuration for unit tests
+------------------------------------
+
+Some of the unit tests require special configuration set as "default" one. This is done automatically by
+automatically adding ``AIRFLOW__CORE__UNIT_TEST_MODE=True`` to the environment variables in Pytest auto-used
+fixture. This in turn makes airflow configuration load test configuration from
+``airflow/config_templates/config_unit_test.yml`` - defaults from the test configuration replace the original
+defaults stored in ``airflow/config_templates/config.yml``. If you want to add some test-only configuration,
+you should copy the option definition to ``config_unit_test.yml`` and change default value there.
```

Comment:
Lots of repetition makes this hard to parse. I'm going to attempt to rephrase it, feel free to take it or now.


```suggestion
Some unit tests require special configurations set as "default". A Pytest auto-use fixture automatically adds ``AIRFLOW__CORE__UNIT_TEST_MODE=True`` to the environment variables to handle this. This, in turn, loads the test's configuration from
``airflow/config_templates/config_unit_test.yml``. Defaults from the test configuration replace the original
defaults stored in ``airflow/config_templates/config.yml``. If you want to add some test-only configuration,
you should copy the option definition to ``config_unit_test.yml`` and change default values there.
```